### PR TITLE
Add Huawei VRP sensors to yaml and fix bug in functions.inc.php

### PR DESCRIPTION
**Pull request by erotel**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by erotel** [comment]
travis check have problem?

**Comment by laf** [comment]
We've not had that issue before. Might be a transient issue, as you've got to make a change to this PR then we'll see if it goes through after.

So, index values. For migrating php code to yaml you need to match them otherwise you lose data. So most of these just need to be '{{ $index }}'.  The exception to this is .1.3.6.1.4.1.2011.5.25.31.1.1.3.1.9. which needs to be 'tx-{{ $index }}'

File: includes/definitions/discovery/vrp.yaml [hunk 1]
```diff
@@ -12,23 +12,31 @@ modules:
...
         dbm:
             data:
                 -
                     oid: hwOpticalModuleInfoTable
                     value: hwEntityOpticalRxPower
                     num_oid: .1.3.6.1.4.1.2011.5.25.31.1.1.3.1.8.
-                    descr: '{{ $entPhysicalName }}'
-                    index: 'hwEntityOpticalRxPower.{{ $index }}'
+                    descr: '{{ $entPhysicalName }} Rx'
+                    index: 'rx-{{ $index }}'
```
Comment (laf) [inline includes/definitions/discovery/vrp.yaml]: This one is just `{{ $index }}`

Comment (erotel) [reply includes/definitions/discovery/vrp.yaml]: old file have `` 'rx-' . $index ``

Comment (laf) [reply includes/definitions/discovery/vrp.yaml]: You're right, when I looked earlier I thought it didn't

**Pull request merged**